<ui version="4.0" >
 <class>ProfitDialog</class>
 <widget class="QDialog" name="ProfitDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Profit margin</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Start date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>End date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="enddateEdit" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>319</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="printpushButton" >
         <property name="text" >
          <string>Print</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProfitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel" >
     <x>324</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
